### ISSUE TITLE
On https://speed.python.org/changes/ logs are not fetched correctly for CPython

### ISSUE THREAD

**Issue by diegorusso** [python/codespeed#48]
Instead a message "Error while retrieving logs: No logs found" is shown.

This is likely caused by this commit https://github.com/python/codespeed/commit/1bdc67a138d75441769c2261045c0a6975e32231 from @vstinner 

`get_scm(project)` returns always None except when the scm is Mercurial.

**Comment by vstinner** [comment]
Ah, someone has to implement a Git class to handle this code path properly.

**Comment by diegorusso** [comment]
Yes, this is what I was thinking. It would be good to wrap git methods in a class like you did for Mercurial.
